db/bigtable_eth1.go

```diff
@@ -3029,6 +3044,13 @@ func (bigtable *Bigtable) GetAddressName(address []byte) (string, error) {
 	ctx, cancel := context.WithDeadline(context.Background(), time.Now().Add(time.Second*30))
 	defer cancel()
```

Comment:
The context is not needed for the `GetEnsNameForAddress` handling below and should therefore be created below it.

Reply:
We usually create the context at the start so I will leave it there.